### PR TITLE
Bump doctrine/dbal to 3.3.8 in master

### PR DESCRIPTION
**Pull request by come-nc**
Signed-off-by: Côme Chilliet <come.chilliet@nextcloud.com>

### CODE REVIEW

**Review by PVince81 — APPROVED** [review]
:+1:

**Review by szaimen — APPROVED** [review]
🐘

File: doctrine/dbal/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Doctrine DBAL
 
-| [4.0-dev][4.0] | [3.1][3.1] | [2.13][2.13] |
-|:----------------:|:----------:|:----------:|
-| [![GitHub Actions][GA 4.0 image]][GA 4.0] | [![GitHub Actions][GA 3.1 image]][GA 3.1] | [![GitHub Actions][GA 2.13 image]][GA 2.13] |
-| [![AppVeyor][AppVeyor 4.0 image]][AppVeyor 4.0] | [![AppVeyor][AppVeyor 3.1 image]][AppVeyor 3.1] | [![AppVeyor][AppVeyor 2.13 image]][AppVeyor 2.13] |
-| [![Code Coverage][Coverage image]][CodeCov 4.0] | [![Code Coverage][Coverage 3.1 image]][CodeCov 3.1] | [![Code Coverage][Coverage 2.13 image]][CodeCov 2.13] |
-| N/A | [![Code Coverage][TypeCov 3.1 image]][TypeCov 3.1] | N/A |
+|                 [4.0-dev][4.0]                  |                     [3.3][3.3]                      |
```
Comment (nickvergessen) [inline doctrine/dbal/README.md:3]: we should not ship this file, please remove it and add to gitignore

File: doctrine/deprecations/README.md [hunk 2]
```diff
@@ -137,6 +137,13 @@ class MyTest extends TestCase
```
Comment (nickvergessen) [inline doctrine/deprecations/README.md:137]: we should not ship this file, please remove it and add to gitignore

File: doctrine/deprecations/phpcs.xml [hunk 3]
```diff
@@ -9,6 +9,8 @@
     <!-- Ignore warnings, show progress of the run and show sniff names -->
```
Comment (nickvergessen) [inline doctrine/deprecations/phpcs.xml:9]: we should not ship this file, please remove it and add to gitignore

File: doctrine/event-manager/README.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # Doctrine Event Manager
 
-[![Build Status](https://travis-ci.org/doctrine/event-manager.svg)](https://travis-ci.org/doctrine/event-manager)
```
Comment (nickvergessen) [inline doctrine/event-manager/README.md:3]: we should not ship this file, please remove it and add to gitignore

File: doctrine/event-manager/UPGRADE.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Upgrade to 1.2
```
Comment (nickvergessen) [inline doctrine/event-manager/UPGRADE.md:1]: we should not ship this file, please remove it and add to gitignore

File: doctrine/event-manager/phpstan.neon.dist [hunk 6]
```diff
@@ -0,0 +1,5 @@
+parameters:
```
Comment (nickvergessen) [inline doctrine/event-manager/phpstan.neon.dist:1]: we should not ship this file, please remove it and add to gitignore

File: doctrine/event-manager/psalm.xml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```
Comment (nickvergessen) [inline doctrine/event-manager/psalm.xml:1]: we should not ship this file, please remove it and add to gitignore

File: psr/cache/README.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+PSR Cache
```
Comment (nickvergessen) [inline psr/cache/README.md:1]: we should not ship this file, please remove it and add to gitignore

File: psr/cache/composer.json [hunk 9]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (nickvergessen) [inline psr/cache/composer.json:1]: we should not ship this file, please remove it and add to gitignore

File: doctrine/event-manager/composer.json [hunk 10]
```diff
@@ -1,50 +1,69 @@
 {
     "name": "doctrine/event-manager",
-    "type": "library",
```
Comment (nickvergessen) [inline doctrine/event-manager/composer.json:3]: we should not ship this file, please remove it and add to gitignore

File: doctrine/deprecations/composer.json [hunk 11]
```diff
@@ -8,9 +8,9 @@
         "php": "^7.1|^8.0"
     },
     "require-dev": {
```
Comment (nickvergessen) [inline doctrine/deprecations/composer.json:10]: we should not ship this file, please remove it and add to gitignore

File: doctrine/dbal/composer.json [hunk 12]
```diff
@@ -32,29 +32,35 @@
     ],
     "require": {
         "php": "^7.3 || ^8.0",
-        "composer/package-versions-deprecated": "^1.11.99",
-        "doctrine/cache": "^1.0|^2.0",
-        "doctrine/deprecations": "^0.5.3",
-        "doctrine/event-manager": "^1.0"
+        "composer-runtime-api": "^2",
```
Comment (nickvergessen) [inline doctrine/dbal/composer.json:35]: we should not ship this file, please remove it and add to gitignore

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Pull request merged**